Test/pruning/Reveal.bpl

```diff
@@ -57,4 +57,13 @@ procedure HideSpecific() {
   hide inner;
   reveal inner;
   assert inner(3) == 42;
+}
+
+revealed function alwaysRevealed(x: int): int uses {
+  hideable axiom (forall x: int :: {alwaysRevealed(x)} alwaysRevealed(x) == 42);
+}
+
+procedure AlwaysRevealed() {
+  hide *;
+  assert alwaysRevealed(3) == 42;
```

Comment:
```suggestion
  hide alwaysRevealed;
  assert alwaysRevealed(3) == 42;
```
Would that work too?

Reply:
Yes, although in Dafny you wouldn't be able to write this, since you can't mark any Dafny functions as being always revealed. Only the prelude functions will get this property.